RedHatInsights/insights-core
# Force TZ=UTC on oscap xccdf eval

To test:

On a system with timezone set to non-UTC:
1. Assign a policy to the host in the compliance UI
2. Run `insights-client --compliance` on the host
3. Ensure the last scanned time in the compliance UI is a few minutes ago

See this OpenSCAP issue: https://github.com/OpenSCAP/openscap/issues/1449

https://projects.engineering.redhat.com/browse/RHICOMPL-450

CC: @dLobatog 

Signed-off-by: Andrew Kofink <akofink@redhat.com>
